packages/ckeditor5-list/src/tododocumentlist/checktododocumentlistcommand.ts

```diff
@@ -0,0 +1,108 @@
+/**
+ * @license Copyright (c) 2003-2023, CKSource Holding sp. z o.o. All rights reserved.
+ * For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-oss-license
+ */
+
+/**
+ * @module list/tododocumentlist/checktododocumentlistcommand
+ */
+
+import { Command, type Editor } from 'ckeditor5/src/core';
+import type { Element } from 'ckeditor5/src/engine';
+import { getAllListItemBlocks } from '../documentlist/utils/model';
+
+const attributeKey = 'todoListChecked';
```

Comment:
Be consistent with letter case. I.e. `ITEM_TOGGLE_KEYSTROKE` in other file is upper-case.